compare_similarity_between_two_texts_using_openai/main.bal

```diff
@@ -5,15 +5,15 @@ import ballerinax/openai.embeddings;
 configurable string openAIToken = ?;
 
 public function main() returns error? {
-    final embeddings:Client openAIEmbeddings = check new ({auth: {token: openAIToken}});
+    final embeddings:Client embeddingsClient = check new ({auth: {token: openAIToken}});
```

Comment:
Let's not change the client naming conventions we've used through out the samples
```suggestion
    final embeddings:Client openAIEmbeddings = check new ({auth: {token: openAIToken}});
```